src/cli/mod.rs

```diff
@@ -0,0 +1,11 @@
+mod config;
+use crate::api_client;
```

Comment:
generally prefer the style to import the "full" thing
```suggestion
use crate::api_client::Client;
```